crates/runtime/src/externref.rs

```diff
@@ -563,10 +593,10 @@ impl VMExternRefActivationsTable {
     /// `insert_slow_path` to infallibly insert the reference (potentially
     /// allocating additional space in the table to hold it).
     #[inline]
-    pub fn try_insert(&self, externref: VMExternRef) -> Result<(), VMExternRef> {
+    pub fn try_insert(&mut self, externref: VMExternRef) -> Result<(), VMExternRef> {
```

Comment:
Nice that we can use `&mut self` down here too, I hadn't realized that we could propagate these changes this deep.